source/avr/pgmspace.h

```diff
@@ -18,8 +18,23 @@
  * conversion of Arduino specific code.
  */
 
+// Not defined in Windows
+#define PROGMEM
+
 typedef const char* PGM_P;
 typedef const void* PGM_VOID_P;
+typedef void PROGMEM prog_void;
```

Comment:
Since you have already `#define PROGMEM`, is it necessary to keep in the other 12 `PROGMEM`s in your `typedef` statements? Legitimate question, I don't know the answer, but I would guess, "No".